package.json

```diff
@@ -11,10 +11,10 @@
     "fs": false
   },
   "dependencies": {
-    "superagent": "3.8.3"
+    "superagent": "3.5.2"
```

Comment:
@ograu @marians: 

This is a bit of a problem in this repo that is getting outdated.

Whenever you generate the client, it will also touch this `package.json` and reduce the version of `superagent` and `mocha`

The lower superagent version it sets has a known security vulnerability, so you should avoid committing these changes in the final merge.